django/db/backends/oracle/creation.py

```diff
@@ -77,9 +77,6 @@ def _create_test_db(self, verbosity=1, autoclobber=False, keepdb=False):
             try:
                 self._create_test_user(cursor, parameters, verbosity, keepdb)
             except Exception as e:
-                # If we want to keep the db, then we want to also keep the user.
-                if keepdb:
-                    return
```

Comment:
Okay, but should this have been part of the original security fix? While it hid a problem, it doesn't seem related.